tests/foreman/cli/test_location.py

```diff
@@ -45,8 +35,10 @@ def test_create_location_with_different_names_positive(self, name):
         @Assert: Location is created successfully and has proper name
 
         """
-        loc = make_location({'name': name})
-        self.assertEqual(loc['name'], name)
+        for name in valid_data_list():
```

Comment:
I had confirmation from devs (as far as I remember from mmcune) that is intended behavior and 246 symbols is a value that used in DB for Orgzanization and Location entities. I can look on that more tomorrow